src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -81,6 +81,58 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers {
 
   }
 
+  it should "list all entity types with their attribute names" in withDefaultTestDatabase {
+
+    withWorkspaceContext(testData.workspace) { context =>
+
+      val desiredTypesAndAttrNames = Map(
+        "Sample" -> Seq("type", "whatsit", "thingies", "quot", "somefoo", "tumortype", "confused", "cycle"),
+        //"Aliquot" -> Seq(), NOTE: this is commented out because the db query doesn't return types that have no attributes.
```

Comment:
In `getAttrNamesAndEntityTypes`, yes. But in the `getEntityTypeMetadata` call the keys get merged, so it'll be returned as an empty `Seq()` (as in the test for that function).